e2e-tests/portal/cypress/e2e/journeys/checker/cannot-submit-a-deal-with-newly-issued-facilities-that-have-cover-start-dates-before-min-submission-date/cannot-submit-a-deal-with-newly-issued-facilities-that-have-cover-start-dates-before-min-submission-date.spec.js

```diff
@@ -1,40 +1,63 @@
-const pages = require('../../../pages');
-const relative = require('../../../relativeURL');
-const theDeal = require('./deal');
+import { add } from 'date-fns';
+
+const { contractConfirmSubmission, contract, bondDetails, dashboardDeals } = require('../../../pages');
 const MOCK_USERS = require('../../../../../../e2e-fixtures');
 
-const { ADMIN, BANK1_MAKER1, BANK1_CHECKER1 } = MOCK_USERS;
+const { ADMIN, BANK1_MAKER1 } = MOCK_USERS;
+
+const tomorrow = add(new Date(), { days: 1 });
+const startDate = {
+  day: tomorrow.getDate(),
+  month: tomorrow.getMonth(),
+  year: tomorrow.getFullYear(),
+};
+
+const endDate = {
+  day: tomorrow.getDate(),
+  month: tomorrow.getMonth(),
+  year: tomorrow.getFullYear() + 1,
+};
 
 // DTFS2-2839
 context(
-  'Checker tries to submit a deal that has changed/newly issued facilities (in `Ready for check` status) with cover start dates that are before MIN submission date',
+  'Maker cannot submit a deal to checker when facility has cover start dates before the MIN submission date" (should see validation errors, submit button should not exist)',
   () => {
-    let deal;
-    let dealId;
-
     before(() => {
       cy.deleteDeals(ADMIN);
 
-      cy.insertOneDeal(theDeal, BANK1_MAKER1).then((insertedDeal) => {
-        deal = insertedDeal;
-        dealId = deal._id;
-      });
+      cy.createBssEwcsDeal({});
     });
 
-    it('should throw error and not submit or redirect', () => {
```

Comment:
Can we add this test back in please

Reply:
We discussed to rename the test/change as the flow is maker cannot submit the deal with cover date before submission date as the validation error are displayed only when entering the cover date before submission date as a make.